### PR TITLE
[FIX] sale_timesheet_line_exclude: user friendly Non-billable label

### PR DESCRIPTION
**Pull request by dreispt**

### CODE REVIEW

**Review by alexey-pelykh — APPROVED** [review]

**Review by marcelsavegnago — APPROVED** [review]

File: sale_timesheet_line_exclude/readme/USAGE.rst [hunk 1]
```diff
@@ -1,11 +1,9 @@
...
+  # Go to *Timesheets > Timesheet > All Timesheets*
+  # On the Timesheet list view, check *Non-billable* for specific timesheet entries
 
 or:
 
-# Go to *Project > All Tasks*
-# Open specific task for editing
-# Switch to *Timesheets* tab
-# Check *Exclude From Billing* for a specific timesheet entry
+  # Go to *Project > All Tasks*, ans open a specific Task form
```
Comment (bodedra) [inline sale_timesheet_line_exclude/readme/USAGE.rst:8]:
```suggestion
  # Go to *Project > All Tasks*, and open a specific Task form
```

Comment (alexey-pelykh) [reply sale_timesheet_line_exclude/readme/USAGE.rst:8]: @dreispt it should be `#.` and without spaces otherwise it indents it even more

Comment (dreispt) [reply sale_timesheet_line_exclude/readme/USAGE.rst:8]: You're right. Actually, a regular `- ` should do: https://docutils.sourceforge.io/docs/user/rst/quickref.html#bullet-lists

Comment (alexey-pelykh) [reply sale_timesheet_line_exclude/readme/USAGE.rst:8]: Let's fix that and merge I guess

Comment (pedrobaeza) [reply sale_timesheet_line_exclude/readme/USAGE.rst:8]: The idea of `#.` is to have a numbered bullet list, so better to stick to this convention than to put `-`

**Review by gurneyalex — APPROVED** [review]

**Pull request merged**